test/Transactions/Orleans.Transactions.Tests/Memory/FaultInjectionConsistencyTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Orleans.TestingHost;
+using Orleans.Transactions.Tests.Consistency;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Orleans.Transactions.Tests.Memory
+{
+    [TestCategory("BVT"), TestCategory("Transactions")]
```

Comment:
I was under the impression that these tests were expected to be used as developer tests initially, with nightly runs to be added later.  Marking them as BVT means they'll be run with every code submission.  If these tests are not expected to succeed consistently, we shouldn't mark them BVT.

Reply:
yep, that was not the intention. I will mark a small subset as BVT.